guides/release/components/glimmer-components-dom.md

```diff
@@ -320,6 +359,7 @@ import action from "@ember/object";
 export default class AudioPlayer extends Component {
   @action
   play() {
+    // TODO
```

Comment:
is this intentional?

Reply:
Yes -- it gets updated with something else later in the documentation. It's a TODO from the perspective of the docs :P